api/src/controllers/dashboard-controller.js

```diff
@@ -0,0 +1,21 @@
+import dashboardService from '../services/dashboard-service';
+
+export default class dashboardController {
+
+    static register(app) {
+
+        app.get('/dashboard/:userId', function (req, res) {
+            const  userId  = req.params['userId'];
+console.log('a')
+console.log(userId, req.params);
+            dashboardService.getUserPosts(userId, function (results) {
+                console.log('results is ', results);
+                if(results.length===0){
+                    res.end(JSON.stringify("No results"));
```

Comment:
Do you like this way of doing it? It seems slightly strange to me because it could just be an empty array and the front end will work out that there weren't any results, wheras in this case the frontend has to do like if(results === 'No results')...